BrushedMotorControllerFirmware/include/driver.c

```diff
@@ -38,7 +38,7 @@ static int set_direction_raw(enum MotorDirection direction, enum ChannelNumber c
 #pragma endregion InternalFunctions
 
 /// CONTROL MODE - whether speed or position is controlled
-static enum ControlModes control_mode = SPEED;
+static enum ControlModes control_mode = (enum ControlModes)0;//Default to first defined control mode
```

Comment:
static  uninitialised variables are always defaulted with `0` value
```suggestion
static enum ControlModes control_mode;
```